common/com/twitter/intellij/pants/util/PantsUtil.java

```diff
@@ -763,7 +763,15 @@ public static boolean supportExportDefaultJavaSdk(@NotNull final String pantsExe
     }
 
     final String defaultPlatform = exportResult.getJvmPlatforms().getDefaultPlatform();
-    final String jdkName = String.format("JDK from pants %s", defaultPlatform);
+
+    // if defaultPlatform is java8, then jdkName should be 1.8_from_pants
+    String jdkName = "1.x_from_pants";
```

Comment:
It would be useful to still include the jdkName in this default name. That way if the loop below didn't match, the name would have the platform info in it.